src/Upgrade/Application/Dto/IntegratorResponseDto.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace Upgrade\Application\Dto;
+
+class IntegratorResponseDto
```

Comment:
strange use of data transfer. if you don't want to use output you can create an adapter.

Reply:
or use set-get for properties

Reply:
You are right, usually, we use some builder for DTO 
In this case, the response structure is so simple, and it's not necessary and I want to follow the same structure as in ComposerLockDiffDto